README.md

```diff
@@ -144,6 +144,10 @@ async function subsystem (fastify, opts) {
 }
 ```
 
+## TypeScript support
+
+To use this module with TypeScript, make sure to install `@types/express`.
```

Comment:
this is already true on master, might wanna land the readme changes while waiting on a decision whether or not this should land at all?